scripts/apps/archive/services/FamilyService.ts

```diff
@@ -53,6 +53,19 @@ export function FamilyService(api, desks) {
         return query(filter, 'versioncreated', 'desc');
     };
 
+    this.fetchLinks = (item: IArticle) => api.query('links', {
```

Comment:
Could you use `dataApi` instead of the `api` service? The problem with the `api` service is that it modifies original objects when patching and drops some fields.

Querying might not be affected by this, but I suggest phasing out the usage of `api` and not use it in new code unless it doesn't work otherwise.